### DISCUSSION TITLE
Does this work on Modified On column?

### DISCUSSION THREAD

**Discussion by lambeaulady** [Innofactor/XrmToolBox.BulkDataUpdater#244] [Q&A]
I attempted to use this to update a Modified On column but it didn't work. Am I doing something wrong or is this not supported?

**Comment by rappen** [comment]
I can't say I know exactly how Dataverse is working, but my best guess (by testing it) is that you _**may**_ change ModifiedOn, but the background is always setting ModifiedOn to `now`, which overrides what we tried to do.
